### PR TITLE
Fix failing build & security warning

### PR DESCRIPTION
**Pull request by donggyu04**
1. Fix failing build
2. Remove unused npm dependencies(`sass` and `scss`) that has security warning too
3. bump OSHI up to 6.1.x to support M1 arch

### CODE REVIEW

File: ngrinder-core/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ description = "ngrinder-core"
 dependencies {
     api (project(":ngrinder-groovy"))
     api (group: "org.apache.commons", name: "commons-compress", version: "1.4.1")
-    api (group: "com.github.oshi", name: "oshi-core", version: "5.2.2")
+    api (group: "com.github.oshi", name: "oshi-core", version: "6.1.6")
```
Comment (donggyu04) [inline ngrinder-core/build.gradle:6]: to support m1

Comment (junoyoon) [reply ngrinder-core/build.gradle:6]: Good!

**Review by junoyoon — APPROVED** [review]

File: ngrinder-frontend/build.gradle [hunk 2]
```diff
@@ -18,4 +18,5 @@ task webpack(dependsOn: "npmInstall", type: NodeTask) {
 task checkES5(type: NpxTask) {
     command = "es-check@$esCheckModuleVersion"
     args = ["es5", "../ngrinder-controller/build/classes/main/static/js/app.js", "--verbose"]
+    ignoreExitValue = true
```
Comment (donggyu04) [inline ngrinder-frontend/build.gradle:21]: IE11 is officially deprecated from 2022.06.15 so, how about we stop supporting IE too?

File: ngrinder-frontend/package.json [hunk 3]
```diff
@@ -22,8 +22,8 @@
     "axios": "^0.21.1",
     "base64-js": "^1.2.1",
     "billboard.js": "^1.11.0",
-    "bootbox": "^5.2.0",
-    "bootstrap": "4.3.1",
+    "bootbox": "^5.5.3",
```
Comment (donggyu04) [inline ngrinder-frontend/package.json:25]: Many of npm dependencies we are using have been deprecated. I didn't fix it in this PR, but it needs to be fixed soon.

File: ngrinder-frontend/build.gradle (hunk 2 above)
Comment (junoyoon) [reply ngrinder-frontend/build.gradle:21]: Yep. stop supporting this.

**Pull request merged**